mat/lq.go

```diff
@@ -19,14 +20,14 @@ type LQ struct {
 	cond float64
 }
 
-func (lq *LQ) updateCond() {
+func (lq *LQ) updateCond(norm lapack.MatrixNorm) {
 	// A = LQ, where Q is orthonormal. Orthonormal multiplications do not change
 	// the condition number. Thus, ||A|| = ||L|| ||Q|| = ||Q||.
```

Comment:
Unrelated to this PR but I think that the formula misuses the norm notation for the condition number. Denoting it for example by `κ`, it should probably read:
`Thus, κ(A) = κ(L*Q) = κ(L)`.
This would be also in line with what the code does. Also, I'm not sure if the statement "Orthogonal multiplications do not change the condition number" is true for the max row/col sum norms used here, but since the exact value of the condition number is not important, we can probably leave the comment.